src/app/shared/session/session.query.ts

```diff
@@ -39,6 +39,7 @@ export class SessionQuery extends Query<SessionState> {
   entryPageTitle$: Observable<string> = this.entryTypeDisplayName$.pipe(map((displayName: string) => displayName + 's'));
   myEntryPageTitle$: Observable<string> = this.entryType$.pipe(map((entryType: EntryType) => 'my ' + entryType + 's'));
   isService$: Observable<boolean> = this.entryType$.pipe(map((entryType) => entryType === EntryType.Service));
+  isAppTool$: Observable<boolean> = this.entryType$.pipe(map((entryType) => entryType === EntryType.Tool));
```

Comment:
I think this echoes one of Kathy's earlier comments, but should this be ` === EntryType.AppTool`? Or should `isAppTool$` be named `isTool$`

Reply:
Yeah no I tried using it with `=== EntryType.AppTool` but that doesn't work, GitHub App tools are being returned as type tool at least from this check

Reply:
maybe it makes more sense to change it to isTool$?

Reply:
> maybe it makes more sense to change it to isTool$?

It sounds like it based on the current code.

Do you have both tools and app tools to try this out?

Reply:
Asking because I don't think your screenshots have any app tools... There should be a "github.com" section under My Tools.

Reply:
No I don't have any app tools, I tested it out by running the `githubAppTool.ts` test locally and using the test ui. There has some app tools and regular tools